### PR TITLE
feature(monitoring): set the scylla monitoring image to 4.8.0

### PR DESCRIPTION
**Pull request by amnonh**
Set the monitoring image to version 4.8.0

#### Testing
- [x] aws provision test
- [x] gce provision test
- [x] check monitor can be retrieved 
- [x] check screenshots working
- [x] 🟢 https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity-10gb-3h-ipv6-test/14/

### CODE REVIEW

File: defaults/aws_config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
 ami_id_loader: 'scylla-qa-loader-ami-v21-ubuntu22'  # versions: c-s 2024.2.0-rc0, s-b 0.1.21
 # manual on updating monitor images see: docs/monitoring-images.md
-ami_id_monitor: 'scylladb-monitor-4-7-2-2024-05-13t08-38-47z'
```
Comment (fruch) [inline defaults/aws_config.yaml:12]: what about the GCP images ?

Comment (fruch) [reply defaults/aws_config.yaml:12]: and also updating the `monitor_branch` default, since in some platform we don't have images, i.e. Azure

**Comment by fruch** [comment]
@amnonh 

The image in AWS isn't found on eu-west-1, is those images synced across regions ?

**Comment by amnonh** [comment]
They should be adding @yaronkaikov, who is responsible for image creation.

**Comment by yaronkaikov** [comment]
> @amnonh
> 
> The image in AWS isn't found on eu-west-1, is those images synced across regions?

It's there

image

**Comment by fruch** [comment]
> > @amnonh
> > The image in AWS isn't found on eu-west-1, is those images synced across regions?
> 
> It's there
> 
> image

@yaronkaikov, we need the Name tag, so we can find it.

**Comment by yaronkaikov** [comment]
> > > @amnonh
> > > The image in AWS isn't found on eu-west-1, is those images synced across regions?
> > 
> > 
> > It's there
> > image
> 
> @yaronkaikov, we need the Name tag, so we can find it.

There is one

image

**Comment by fruch** [comment]
> > > > @amnonh
> > > > The image in AWS isn't found on eu-west-1, is those images synced across regions?
> > > 
> > > 
> > > It's there
> > > image
> > 
> > 
> > @yaronkaikov, we need the Name tag, so we can find it.
> 
> There is one
> 
> image

o.k. the code in SCT is looking up by name, only look in one account, and not in the images account.
that part was used for fetching other images, and not scylla images by name

**Comment by fruch** [comment]
@yaronkaikov 

a PR to fix it, and lookup on both accounts, so monitor image can be found:
https://github.com/scylladb/scylla-cluster-tests/pull/8232

previously the image were clone into the main account (since they were available on all regions)
we should stop doing that..., since they now should be copied into all regions to begin with.

**Comment by amnonh** [comment]
I've recreated 4.8.0 this morning, we should re-generate the images

**Comment by yaronkaikov** [comment]
> I've recreated 4.8.0 this morning, we should re-generate the images

Why re-creating? now we would have two images which are 4.8.0,

**Comment by amnonh** [comment]
@yaronkaikov 4.8.0 is in RC mode, and I made changes, so I had to recreate the release, I'm not sure if images were created.

**Comment by yaronkaikov** [comment]
> @yaronkaikov 4.8.0 is in RC mode, and I made changes, so I had to recreate the release, I'm not sure if images were created.

@amnonh So let's call it (tag name) 4.8.0-rcX, and then you can increase the number for every change until it's officially out. This way QA for example can distinguish between each image by name

**Comment by amnonh** [comment]
@yaronkaikov image names contain the date, I can add an rcX to the tags, if it's supported

**Comment by yaronkaikov** [comment]
> @yaronkaikov image names contain the date, I can add an rcX to the tags, if it's supported

Thanks, it should be ok, as we do in Scylla
image

**Comment by fruch** [comment]
I've found one more issue with the fact the images are on the releng account, and that they are not public https://github.com/scylladb/scylla-cluster-tests/pull/8258

one that would get reviewed and merged, we can do one more round 

@amnonh did you created new images ?

**Comment by amnonh** [comment]
I did:
AWS: scylladb-monitor-4-8-0-2024-08-06t03-34-43z
GCP: scylladb-monitor-4-8-0-2024-08-06t03-34-43z

**Comment by fruch** [comment]
> I did: AWS: scylladb-monitor-4-8-0-2024-08-06t03-34-43z GCP: scylladb-monitor-4-8-0-2024-08-06t03-34-43z

@amnonh @yaronkaikov  those images aren't working...

```
botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-02f241d7e29ae98ad]
```

**Comment by fruch** [comment]
> > I did: AWS: scylladb-monitor-4-8-0-2024-08-06t03-34-43z GCP: scylladb-monitor-4-8-0-2024-08-06t03-34-43z
> 
> @amnonh @yaronkaikov those images aren't working...
> 
> ```
> botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-02f241d7e29ae98ad]
> ```

meanwhile I manually setup the sharing, and raised:
https://github.com/scylladb/scylla-monitoring/issues/2375 

for the next time we need those images

**Review by fruch — APPROVED** [review]
LGTM

**Comment by mikliapko** [comment]
@amnonh @fruch 
I suppose this fix broke some Manager tests:
```
13:01:34  ----- LAST ERROR EVENT -------------------------------------------------------
13:01:34  2024-08-13 11:01:20.621: (TestFrameworkEvent Severity.ERROR) period_type=one-time event_id=8e220cb6-4ea4-4484-9223-5099475f54c2, source=MgmtCliTest.SetUp()
13:01:34  exception=An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-0b02bf09acc3d3292]
```
https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/ubuntu22-installation-test/2/

**Comment by fruch** [comment]
> @amnonh @fruch 
> I suppose this fix broke some Manager tests:
> ```
> 13:01:34  ----- LAST ERROR EVENT -------------------------------------------------------
> 13:01:34  2024-08-13 11:01:20.621: (TestFrameworkEvent Severity.ERROR) period_type=one-time event_id=8e220cb6-4ea4-4484-9223-5099475f54c2, source=MgmtCliTest.SetUp()
> 13:01:34  exception=An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-0b02bf09acc3d3292]
> ```
> https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/ubuntu22-installation-test/2/

In which region is this running?

I can guess I've missed some AMI, setting the sharing information

Until https://github.com/scylladb/scylla-monitoring/issues/2375
Would be fixed

**Comment by mikliapko** [comment]
> > @amnonh @fruch
> > I suppose this fix broke some Manager tests:
> > ```
> > 13:01:34  ----- LAST ERROR EVENT -------------------------------------------------------
> > 13:01:34  2024-08-13 11:01:20.621: (TestFrameworkEvent Severity.ERROR) period_type=one-time event_id=8e220cb6-4ea4-4484-9223-5099475f54c2, source=MgmtCliTest.SetUp()
> > 13:01:34  exception=An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-0b02bf09acc3d3292]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/ubuntu22-installation-test/2/
> 
> In which region is this running?
> 
> I can guess I've missed some AMI, setting the sharing information
> 
> Until [scylladb/scylla-monitoring#2375](https://github.com/scylladb/scylla-monitoring/issues/2375) Would be fixed

It's us-east-1 region

**Comment by fruch** [comment]
> > > @amnonh @fruch
> > > I suppose this fix broke some Manager tests:
> > > ```
> > > 13:01:34  ----- LAST ERROR EVENT -------------------------------------------------------
> > > 13:01:34  2024-08-13 11:01:20.621: (TestFrameworkEvent Severity.ERROR) period_type=one-time event_id=8e220cb6-4ea4-4484-9223-5099475f54c2, source=MgmtCliTest.SetUp()
> > > 13:01:34  exception=An error occurred (AuthFailure) when calling the RequestSpotInstances operation: Not authorized for images: [ami-0b02bf09acc3d3292]
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/ubuntu22-installation-test/2/
> > 
> > 
> > In which region is this running?
> > I can guess I've missed some AMI, setting the sharing information
> > Until [scylladb/scylla-monitoring#2375](https://github.com/scylladb/scylla-monitoring/issues/2375) Would be fixed
> 
> It's us-east-1 region

Fix the permissions, and re-run that job

**Pull request merged**